www/docs/guides/what_s_starknet.js.md

```diff
@@ -23,6 +23,8 @@ Some important topics that have to be understood:
   - private customized version of Starknet.
   - local Starknet node (connected to mainnet or testnet).
 
+> Understand what is Starknet and how it works is necessary. Then, you can learn how to interact with it using Starknet.js. So, at this stage, you should be aware of the content of the [Starknet official doc](https://docs.starknet.io/documentation/)
```

Comment:
we could add also https://github.com/starknet-edu/starknetbook

Reply:
Implemented